tests/zquantum/qaoa/estimators/gibbs_estimator_test.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+from openfermion import QubitOperator, IsingOperator
+
+from zquantum.core.interfaces.mock_objects import (
+    MockQuantumBackend,
+)
+from zquantum.core.circuits import Circuit, X, H
+from zquantum.core.measurement import Measurements
+from zquantum.qaoa.estimators import GibbsEstimator
+
+from zquantum.core.interfaces.estimation import EstimationTask
+import numpy as np
+
+
+class TestGibbsEstimator:
+    @pytest.fixture(params=[1.0, 0.8, 0.5, 0.2])
+    def estimator(self, request):
+        return GibbsEstimator(alpha=request.param)
+
+    @pytest.fixture()
+    def circuit(self):
+        return Circuit([X(0)])
+
+    @pytest.fixture()
+    def operator(self):
+        return IsingOperator("Z0")
+
+    @pytest.fixture()
+    def estimation_tasks(self, operator, circuit):
+        return [EstimationTask(operator, circuit, 10)]
+
+    @pytest.fixture()
+    def backend(self, request):
+        backend = MockQuantumBackend()
+
+        def custom_run_circuit_and_measure(circuit, n_samples):
```

Comment:
We've recently updated how `MockQuantumBackend` works, this whole function is no longer needed, you can just do: `return MockQuantumBackend()`.